mobile/models/partner.coffee

```diff
@@ -10,6 +10,15 @@ PartnerShows = require '../collections/partner_shows.coffee'
 fetchUntilEnd = require('artsy-backbone-mixins').Fetch().fetchUntilEnd
 Relations = require './mixins/relations/partner.coffee'
 
+INSTITUTION        = 'institution'
+GALLERY_DEFAULT    = 'gallery_default'
+GALLERY_DEPRECATED = 'gallery_deprecated'
+GALLERY_ONE        = 'gallery_one'
+GALLERY_TWO        = 'gallery_two'
+GALLERY_THREE      = 'gallery_three'
+
+PARTNER_TYPES = [ INSTITUTION, GALLERY_ONE, GALLERY_TWO, GALLERY_THREE ]
```

Comment:
#minor. I would suggest we rename the constants to be more self-explanatory. Something like
```coffeescript
ACTIVE_PARTNER_LAYOUTS = [ ... ]
```

And the usage becomes more clear:
```coffeescript
unless partner.get('profile_layout') in ACTIVE_PARTNER_LAYOUTS
  # do something...
```